src/conf_mode/protocols_static.py

```diff
@@ -98,6 +98,15 @@ def generate(static):
     return None
 
 def apply(static):
+    ## Put routing table names in /etc/iproute2/rt_tables
+    with open("/etc/iproute2/rt_tables.d/vyos.conf", 'w') as f:
+        print("Generated by VyOS (protocols_static.py), do not edit by hand", file=f)
```

Comment:
Missing leading `#` to indicate a comment - could probably fuck things up?

Any reason to not use JInja2 and `render()` here? I know it's more effort but it's also more consistent.

Reply:
Good catch! That issue wasn't in my test script, I'm not sure how I introduced it...

As of Jinja2, I thought since this is very simple and has no potential to ever grow, it will neither cause problems nor benefit from Jinja2.